cpp/test/Ice/slicing/objects/AllTests.cpp

```diff
@@ -686,25 +681,20 @@ class PreservedI : public virtual Preserved
 
 int PreservedI::counter = 0;
 
-#ifndef ICE_CPP11_MAPPING
+// TODO: this class is not used anywhere
 class PreservedFactoryI : public Ice::ValueFactory
```

Comment:
pretty sure these are from the gc tests.

Reply:
Removed.